aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,69 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[When to use/not to use](#use)</br>
+[Do’s and Don’ts visual examples](#dos)</br>
```

Comment:
These two comments: 1) rely on markdown generated ids, and 2) use either return or 2 spaces + return can be applied throughout.

Reply:
huh? 😂